037-WindowsVirtualDesktop/Coach/README.md

```diff
@@ -0,0 +1,20 @@
+# What The Hack - Windows Virtual Desktop
+
+## Introduction
+
+Welcome to the Coach's guide for the Windows Virtual Desktop What the Hack. Below are links to each challenge that has specific guidance for each challenge. 
+
+## Coach's Guides
+
+- Challenge 0: **[Windows Virtual Desktop  Pre-Reqs](00-Pre-Reqs.md)**
+- Challenge 1: **[Planning your WVD Architecture](01-Plan-WVD-Architecture.md)**
+- Challenge 2: **[Implementing and Managing your WVD Network](02-Implement-Manage-Network.md)**
+- Challenge 3: **[Implement and Manage your storage for FsLogix](03-Implement-Manage-Storage.md)**
+- Challenge 4: **[Create and Manage your Images for your Session Hosts](04-Create-Manage-Images.md)**
+- Challenge 5: **[Create and Configure your WVD HostPools](05-Create-Configure-HostPools.md)**
+- Challenge 6: **[Implement and Manage your FsLogix](06-Implement-Manage-FsLogix.md)**
+- Challenge 7: **[Install and Configure your applications](07-Install-Configure-Apps.md)**
+- Challenge 8: **[Plan and Implement BCDR for your WVD Environment](08-Plan-Implement-BCDR.md)**
+- Challenge 9: **[Automation of your WVD Tasks](09-Automate-WVD-Tasks.md)**
+- Challenge 10: **[Monitor and Manage the performance health of your environment](10-Monitor-Manage-Performance-Health.md)**
+- Challenge 11: **[Configure settings for user experience](11-Configure-User-Experience-Settings.md)**
```

Comment:
Make sure this ToC text matches the headers in the Coach guide for each challenge and that same text should be used in the Student Guide.